src/Project/BuilderCfgFile.ts

```diff
@@ -150,7 +150,7 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
     let inputModel = path.basename(importTFlite.inputPath);
     importTFlite.name = 'ImportTFlite ' + inputModel;
 
-    console.log('importTFlite = ', importTFlite);
+    Logger.debug(this.tag, 'importTFlite = ', importTFlite);
```

Comment:
![image](https://user-images.githubusercontent.com/63768207/170210135-78d30c2a-3366-4ff9-8a54-62bbf0b9dd8d.png)

Reply:
importTflite is object type.